scripts/release.sh

```diff
@@ -18,7 +18,6 @@
 # specific language governing permissions and limitations
 # under the License.
 #
```

Comment:
Do you need to remove `set -Eeuo pipefail`?

Reply:
Yes, because there's a return function here https://github.com/eHealthAfrica/aether/pull/430/files#diff-0d28f82f2d1cc0deb12146b212778bc9R125

Reply:
Maybe you only want to remove the `e` option... :woman_shrugging: 

https://www.gnu.org/software/bash/manual/bash.html#The-Set-Builtin

_This is a personal doubt, not a request or a blocker_